PIM/UserInterface/04_DataTypeList.md

```diff
@@ -0,0 +1,615 @@
+- Sale unit
+
+### Configuration
+
+This data type has no further configuration settings.
+
+
+## Simple price field
+
+![Simple price field](/Assets/Screenshots/PIM/DataTypes/SimplePriceField.png "[Simple price field]")
+
+The data type *Simple price field* is used to indicate a price. You can configure the use of gross or net price, the display of an MSRP field and promotion prices and the currency to be used. Additionally, you can create scale prices in the product view. Unlike the data type *Preise*, the data type *Simple price field* requires less information to indicate the price.
+
+### Configuration
+
+![Simple price field](/Assets/Screenshots/PIM/DataTypes/Configuration/SimplePriceField.png "[Simple price field]")
+
+- ![Toggle](/Assets/Icons/Toggle.png "[Toggle]") *Gross prices*   
+  Enable this toggle to display gross prices. Disable the toggle to display net prices. By default, this toggle is enabled.
+
+  > [Info] Note that it is especially for the ETL-mapping very important if prices are net or gross.
+
+- ![Toggle](/Assets/Icons/Toggle.png "[Toggle]") *Show MSRP field*   
+  Enable this toggle to display the MSRP field. Disable the toggle to hide the MSRP field. By default, this toggle is disabled.
+
+- ![Toggle](/Assets/Icons/Toggle.png "[Toggle]") *Use Promotions*    
+  Enable this toggle to enable to enter a promotion period for prices. Disable the toggle to not allow promotion prices. By default, this toggle is disabled.
+
+- *Currency*   
+  Click the drop-down list to select the currency to be used. All available currencies are displayed in the drop-down list. By default, the default currency is preselected.
+
+[comment]: <> (Not working -> MSRP field is displayed even when the toggle is disabled -> to be fixed)
+
+
+## Stock Value
+
+![Stock Value](/Assets/Screenshots/PIM/DataTypes/StockValue.png "[Stock Value]")
+
+The data type *Stock Value* is used to indicate the number of stocks. You can configure the number of decimal places. A new target channel for this stock is created in the *Stock* module for the inventory assignment. In the *PIM Basic Set*, the following attributes have the data type *Stock Value*:
+- Stock  
+
+### Configuration
+
+![Stock Value](/Assets/Screenshots/PIM/DataTypes/Configuration/StockValue.png "[Stock Value]")
+
+- *Precision*
+  Enter the number of decimals for the stock value. By default the number of decimals is set to **4**.  
+
+- ![Toggle](/Assets/Icons/Toggle.png "[Toggle]") *Read only*
+  Enable this toggle to set the stock value read-only. Disable the toggle to set the stock value editable. By default, this toggle is enabled.
+
+  > [Info] If the warehouse logistic for an product is active and the *Invoicing* module is correctly connected, the stocks are transferred from the invoicing and automatically written into this attribute value. It is only recommended to deactivate the toggle *Read only* if the warehouse logistic is inactive and a stock value is needed.
```

Comment:
It is not the Invoicing module that has to be connected corretly, but the warehousing module